build.gradle

```diff
@@ -41,6 +47,8 @@ buildscript {
         maven { url "https://plugins.gradle.org/m2/" }
         maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
         maven { url "https://d1nvenhzbhpy0q.cloudfront.net/snapshots/lucene/" }
+        maven { url "https://build.shibboleth.net/nexus/content/groups/public" }
```

Comment:
The shibboleth repositories aren't needed after the reversion, but no harm in having them either.